<!-- Collision -->
<element name="collision" required="*">
  <attribute name="name" type="string" default="__default__" required="1"/>
  <attribute name="laser_retro" type="double" default="0" required="0"/>

  <element name="max_contacts" type="int" default="10" required="0"/>
  <element name="mass" type="double" default="0" required="0"/>

  <element name="origin" required="0">
    <attribute name="pose" type="pose" default="0 0 0 0 0 0" required="1"/>
  </element> <!-- End Origin -->

  <include filename="geometry.sdf" required="1"/>
  <include filename="surface.sdf" required="0"/>

</element> <!-- End Collision -->
